automation/run-integration-tests.sh

```diff
@@ -109,4 +113,5 @@ dump_network_info
 install_nmstate
 add_extra_networks
 dump_network_info
+install_dhcpd
```

Comment:
This should be part of the docker image.
But I guess we can change it later.

Reply:
Without it, the CI of this PR will fail, so we could change that later.